web/packages/shared/components/MenuLogin/types.ts

```diff
@@ -48,6 +48,14 @@ export type MenuLoginProps = {
   width?: string;
   ButtonComponent?: ComponentType<ComponentPropsWithRef<typeof ButtonBorder>>;
   buttonText?: string;
+  /*
+   * Text label, instead of default search and filter input,
+   * is shown if this value is set to true.
+   */
+  disableSearchAndFilter?: boolean;
```

Comment:
From what I see, `MenuLogin` doesn't seem to use the term "search" anywhere, other than as the default placeholder when `inputType` is set to filter.

But disregarding the name of this prop, I think it'd nicer to introduce a third option to `MenuInputType` called `NONE` or something. Otherwise it's possible to pass both `disableSearchAndFilter={true}` and `inputType={MenuInputType.FILTER}`, but those two options together don't make much sense.

Reply:
Using `MenuInputType.NONE` is much much better. Thanks https://github.com/gravitational/teleport/pull/52877/commits/a482c40b896bb187e784220fc0366060eaad2dee